modules/manage/partials/troubleshooting.adoc

```diff
@@ -0,0 +1,120 @@
+//tag::deployment[]
+=== A Redpanda Enterprise Edition license is required
+
+During a Redpanda upgrade, if enterprise features are enabled and a valid Enterprise Edition license is missing, Redpanda logs a warning and aborts the upgrade process on the affected broker. This issue prevents a successful upgrade.
+
+If you encounter this issue, follow these steps to recover:
+
+. Downgrade the affected broker to the original version.
+. Do one of the following:
+- xref:get-started:licensing/add-license-redpanda/index.adoc[Apply a valid Redpanda Enterprise Edition license] to the cluster.
+- Disable Enterprise features.
++
+If you do not have a valid license and want to proceed without using Enterprise features, you can disable the enterprise features in your Redpanda configuration.
+
+. Retry the upgrade.
+
+ifndef::env-kubernetes[]
+If you cannot downgrade the affected broker, Redpanda provides a temporary mechanism to bypass this issue and progress the upgrade:
+
+. Use the `RP_FALLBACK_ENTERPRISE_LICENSE` environment variable to inject a temporary license for Redpanda to use during startup. This license is used only for validation during startup and is not persisted.
+
+. Retry the upgrade.
+
+. After the upgrade is complete, persist the license:
++
+[,bash]
+----
+rpk cluster license set <license-key>
+----
+endif::[]
```

Comment:
We were planning not to document this escape hatch and only have it available for support/core engineers. Customers should use the primary recovery mechanism of downgrading and installing a license. cc @mattschumpert 

```suggestion
```